tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -124,7 +124,7 @@ public SecuritySettings(IConfigurationSource? source, IConfigurationTelemetry te
             }
 
             ApiSecurityEnabled = config.WithKeys(ConfigurationKeys.AppSec.ApiSecurityEnabled, "DD_EXPERIMENTAL_API_SECURITY_ENABLED")
-                                       .AsBool(false);
+                                       .AsBool(true);
```

Comment:
Is this feature still "experimental"? It seems weird to me to have a `DD_EXPERIMENTAL_*` feature enabled by default.

Reply:
Never mind. I missed that there we are reading two configs:
- `ConfigurationKeys.AppSec.ApiSecurityEnabled` (not "experimental")
- `"DD_EXPERIMENTAL_API_SECURITY_ENABLED"` as fallback for backwards compat

Reply:
You'll want to fix the comment here, though: https://github.com/DataDog/dd-trace-dotnet/blob/d09331a2d718bd11b3a51fce19d1eaba1884f909/tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.AppSec.cs#L114-L117

Reply:
fixed in [924b6fa](https://github.com/DataDog/dd-trace-dotnet/pull/6043/commits/924b6fa8cead878441d2613fdcddd5a49d748197)